packages/portal/src/pages/item/_.vue

```diff
@@ -356,27 +357,207 @@
       },
 
       async fetchMetadata() {
+        const params = {};
+        if (this.translatingMetadata) {
+          params.profile = 'translate';
+          params.lang = this.metadataLanguage;
```

Comment:
They will be the same value if translatingMetadata is true, so either will work here.

Reply:
Yes, they both work. I'm just thinking that conceptually it would be more correct to use `translationLanguage`. `translationLanguage` is defined by the lang param only, whereas `metadataLanguage` has the UI locale as fallback, which should never be passed as lang param (right?). At the moment it never will, due to the current logic. I think it would help though, to use the specific language prop to keep the difference between `translationLanguage` and `metadataLanguage` clear. I had to untangle that difference a little while reviewing and would like to limit any future confusion.

Reply:
(That also applies for my other similar comment)

Reply:
Agreed, and made it so.